nuxeo-runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/AbstractRuntimeService.java

```diff
@@ -137,13 +145,35 @@ public synchronized void start() {
         if (isStarted) {
             return;
         }
+
+        manager = createComponentManager();
+        try {
+            loadConfig();
+        } catch (IOException e) {
+            throw new RuntimeServiceException(e);
+        }
+
         if (Boolean.parseBoolean(getProperty(REDIRECT_JUL, "true"))) {
             Level threshold = Level.parse(getProperty(REDIRECT_JUL_THRESHOLD, "INFO").toUpperCase());
             JavaUtilLoggingHelper.redirectToApacheCommons(threshold);
         }
+        if (!Boolean.parseBoolean(getProperty(LOG4J_WATCH_DISABLED, "false"))) {
+            long delay;
+            try {
+                delay = Long.parseLong(getProperty(LOG4J_WATCH_DELAY, Long.toString(LOG4J_WATCH_DELAY_DEFAULT)));
+            } catch (NumberFormatException e) {
+                delay = LOG4J_WATCH_DELAY_DEFAULT;
+            }
+            if (Log4JHelper.configureAndWatch(delay * 1000)) {
+                log.info("Configured log4j.xml change detection with a delay of " + delay + "s");
+            } else {
+                log.info("Failed to configure log4j.xml change detection");
```

Comment:
Will the log work fine in that case? Add a Syserr print too or instead?
`info` level instead of `error`?